MODULE.bazel

```diff
@@ -47,33 +47,21 @@ IO_GRPC_GRPC_JAVA_ARTIFACTS = [
 # GRPC_DEPS_END
 
 bazel_dep(name = "bazel_skylib", version = "1.7.1")
-bazel_dep(name = "googleapis", repo_name = "com_google_googleapis", version = "0.0.0-20240326-1c8d509c5")
+bazel_dep(name = "googleapis", repo_name = "com_google_googleapis", version = "0.0.0-20240819-fe8ba054a")
 # CEL Spec may be removed when cncf/xds MODULE is no longer using protobuf 27.x
 bazel_dep(name = "cel-spec", repo_name = "dev_cel", version = "0.15.0")
+bazel_dep(name = "envoy_api", version = "0.0.0-20241214-918efc9")
```

Comment:
We can't upgrade protobuf beyond 23.1 for a good while yet because of the new protobuf-java major version (we support the newer version, but we need to support the older version too). So we can't use this dependency.

The same problem impacts the `xds` dependency.

Reply:
Then you won't be able to support Bazel 8+ which moved java rules to rules_java which in turn uses a newer Protobuf version and java proto rules to Protobuf itself which is also only available since a recent version.

Reply:
I've already tried compiling with Bazel 8 and didn't see any problem. If rules_java requires the newer Protobuf, then yeah, we can't use it.

Reply:
How long do you plan to support Protobuf v23 support?

Reply:
The ecosystem is evolving quickly. Would it be an option that people who want to use an old version of protobuf having to use an old version of other dependencies like grpc-java as well? Then grpc-java on head can move on to newer versions of dependencies.

Reply:
Not really, because then we'd need to support the older version of gRPC longer and it'd impact downstreams as well. It also makes upgrading harder because you have to upgrade many things as once. Because protobuf is releasing incompatible versions doesn't mean grpc-java should take on the extra burden.

We do push Bazel faster than we do other things, because it is changing so much. But the protobuf compatibility issues have been a giant mess without adding the Bazel issues on top.